ui/src/components/NodePartitionTable.js

```diff
@@ -116,7 +117,7 @@ const NodePartitionTable = ({ instanceIP }: { instanceIP: string }) => {
             return getNodePartitionsTableData(
               nodeFSUsageResult.data.result,
               nodeFSSizeResult.data.result,
-              alerts,
+              removeWarningAlerts(alerts),
```

Comment:
I think it is not necessary to filter alerts here

Reply:
Agree. Dropped this commit.